db/data/20240527121747_set_up_sms_reminder_india_2024_june_july.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+class SetUpSmsRemindersIndiaJuneJuly2024 < ActiveRecord::Migration[6.1]
+  INCLUDED_FACILITY_SLUG = [
+    Facility.where(district: "Chennai").pluck(:slug),
+    Facility.where(state: "West Bengal").pluck(:slug)
+  ].flatten
+
+  # Regions where Simple is active and we send sms reminders currently:
+  # States - All of West Bengal, selcted districts in Tamil Nadu
+  # Districts - Chennai
+  REGION_FILTERS = {"facilities" => {"include" => INCLUDED_FACILITY_SLUG}}
+  PATIENTS_PER_DAY = 20_000
+  EXPERIMENTS_DATA = (6..7).map do |month_number|
+    month = Date::ABBR_MONTHNAMES[month_number]
+    {
+      start_time: DateTime.new(2024, month_number).beginning_of_month,
+      end_time: DateTime.new(2024, month_number).end_of_month,
+      current_patients_experiment_name: "Current Patient #{month} 2024",
+      stale_patients_experiment_name: "Stale Patient #{month} 2024"
+    }
+  end
```

Comment:
[nitpick] This is correct, but the number bounds and ranges make me a bit nervous (have to look up always to remember which sides are inclusive and if months start from zero or one).

This is how we have done it in the last couple of these data migrations:
https://github.com/simpledotorg/simple-server/blob/9797949b44290be14a8a278500ef2652f8ab8faf/db/data/20240425072432_set_up_sms_reminders_bd_may_oct_2024.rb#L4-L10

Reply:
This makes sense. Applying the change 👍